RFC.rst

```diff
@@ -62,8 +62,13 @@ to fairly and consistently applying these principles to every aspect of
 managing this project. Project maintainers who do not follow or enforce
 the Code of Conduct may be permanently removed from the project team.
 
-This Code of Conduct applies both within project spaces and in public
-spaces when an individual is representing the project or its community.
+This Code of Conduct applies in all project spaces, as well as in other
+spaces where a user is acting in an official role as a spokesperson for
+the project.
+
+It does not apply to speech which does not pertain to be on behalf of the
+project (for example, opinions shared on social media which are not
+posted on an official project account).
```

Comment:
I think this becomes a tricky issue. Is it the responsibility of the project to police the behavior of its members when they are not acting within the purview of the project? Maybe a good analog for this school bullying. When bullying occurs outside of the school and not in relation to it, when does the school consider it their responsibility to step in and take control the situation?